### PR TITLE
Fix Deprecation warning for PHP 8.2

### PR DESCRIPTION
**Pull request by aviator-ua**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documented?   | no
| Fixed tickets | 
| License       | MIT

Fix Deprecation warnings for PHP 8.2

### CODE REVIEW

**Review by rooodik — APPROVED** [review]

**Comment by Vincz** [comment]
Hey @aviator-ua 
Tests are breaking because of the profile.
Would you mind to update your PR with the profiler fix from here: https://github.com/overblog/GraphQLBundle/commit/e630664b4fd1b1458b95f574564de3b504e740f6#diff-63d69b023b2cc52ebd4f6007144f0146988d1c136b2be997b746707bb098087e
It should fix the tests and I'll merge it after that. 
Thanks

File: src/Controller/ProfilerController.php [hunk 1]
```diff
@@ -52,6 +53,10 @@ public function __invoke(Request $request, string $token): Response
         $this->profiler->disable();
 
         $profile = $this->profiler->loadProfile($token);
+        // Type hint as int for the $limit argument of the find method was updated in Symfony 5.4.22 and 6.2.8
```
Comment (Vincz) [inline src/Controller/ProfilerController.php:56]: Hi @aviator-ua.
Just add a `@phpstan-ignore-line` above line 57 and we can merge.
Thanks!

**Review by Vincz — APPROVED** [review]

**Comment by Vincz** [comment]
Thx @aviator-ua!

**Comment by aviator-ua** [comment]
@Vincz Will you create a tag? Or I should use `dev-0.15` in my project?

**Comment by Vincz** [comment]
@aviator-ua https://github.com/overblog/GraphQLBundle/releases/tag/v0.15.3

**Comment by glye** [comment]
Hi @aviator-ua & @Vincz! It looks like we need this fix in the stable branches 0.13 / 0.14, as well as 1.0.

Could the project README be outdated? It says 1.0 is a dev branch, does not mention 0.15, and says 0.13 / 0.14 are stable branches. But I notice 0.15 got this fix while 0.13 / 0.14 didn't.
Thank!

**Comment by rooodik** [comment]
> Hi @aviator-ua & @Vincz! It looks like we need this fix in the stable branches 0.13 / 0.14, as well as 1.0.
> 
> Could the project README be outdated? It says 1.0 is a dev branch, does not mention 0.15, and says 0.13 / 0.14 are stable branches. But I notice 0.15 got this fix while 0.13 / 0.14 didn't. Thanks!

The fix for 0.14 is done, @Vincz could you have a look, please?

**Pull request merged**